File: js/sumPairs.js
exports.sumPairs = function(arr, num) {
  let output = [];
  for (let i = 0; i < arr.length; i++) {
    for (let j = i; j < arr.length; j++) {
      if (j === i) {
        continue;
      }
      if (arr[i] + arr[j] == num) {
        output.push([arr[i], arr[j]])
      } 
    }
  }

  return output.length === 0 ? 'unable to find pairs' : output;
};

File: python/sum_pairs.py
def sum_pairs(arr, num):
    output = []
    for i in range(0, len(arr)):
        for j in range(i, len(arr)):
            if (i == j):
                continue
            if (arr[i] + arr[j] == num):
                output.append([arr[i], arr[j]])
    return output if len(output) > 0 else 'unable to find pairs'